changelog.d/759.change.rst

```diff
@@ -0,0 +1 @@
+Let ``evolve()`` work with nested ``attrs`` classes.  #634
```

Comment:
```suggestion
``attrs.evolve()`` now works recursively with nested ``attrs`` classes.
```

If you want to link 634 too, you have to created another snippet with the same content. Unfortunately that's the towncrier way.